includes/definitions/teleste.yaml

```diff
@@ -0,0 +1,12 @@
+os: teleste
```

Comment:
teleste appears to be to the vendors name rather than the OS these devices run. Please use the OS name for the product or if wasn't available then something referencing the product itself or product range would be better.